Resources/public/js/views/services/ez-roleserversideviewservice.js

```diff
@@ -22,11 +22,27 @@ YUI.add('ez-roleserversideviewservice', function (Y) {
     Y.eZ.RoleServerSideViewService = Y.Base.create('roleServerSideViewService', Y.eZ.ServerSideViewService, [], {
         initializer: function () {
             this.on('*:contentDiscover', function (e) {
-                e.config.contentDiscoveredHandler = Y.bind(this._assignRole, this);
+                // If we're not discovering for a role assignment, the contentDiscoveredHandler is already set
+                if (this._isDiscoveringForAssigningRole(e)) {
```

Comment:
is it possible to add an extra parameter to the event facade  to say if we are doing a role assignment or not to avoid trying to figure it out ?